.gitattributes

```diff
@@ -1,18 +0,0 @@
-# remove files from deployment using `git archive`
```

Comment:
You may already know this but the PR can't be merged in its current state because you've removed many files -- especially, the other themes -- that aren't related to your goal.

I added this comment to prevent an unintentional merge.